sparkmagic/sparkmagic/utils/utils.py

```diff
@@ -53,6 +56,29 @@ def coerce_pandas_df_to_numeric_datetime(df):
             except (ValueError, TypeError):
                 pass
 
+
+def records_to_dataframe(records_text, kind):
```

Comment:
I'm noticing that there no tests for records_to_dataframe, but only for the sub-routine coerce_pandas_df_to_numeric_datetime. The latter is tested extensively, but not the former.